### PR TITLE
feat: `keypairs` table added and necessary endpoints exposed via accounts api

### PR DESCRIPTION
**Pull request by saledjenic**
Added `keypairs` table which will keep track of migrated keypairs and their state.

### CODE REVIEW

File: multiaccounts/keypairs/database.go [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+			keycard_locked, 
+			account_address, 
+			key_uid) 
+		VALUES
+			(?, ?, ?, ?, ?);
+	`)
+	if err != nil {
+		return err
+	}
+
```
Comment (richard-ramos) [inline multiaccounts/keypairs/database.go:133]:
```suggestion
defer insert.Close()
```

File: multiaccounts/keypairs/database.go [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+func (kp *KeyPairs) UpdateKeyPairName(kcUID string, kpName string) (err error) {
+	update, err := kp.db.Prepare(`
+		UPDATE keypairs 
+		SET 
+			keypair_name = ?
+		WHERE 
+			keycard_uid = ?
+	`)
+	_, err = update.Exec(kpName, kcUID)
```
Comment (richard-ramos) [inline multiaccounts/keypairs/database.go:159]:
```suggestion
        if err != nil {
            return err
        }
        defer update.Close()
	_, err = update.Exec(kpName, kcUID)
```

File: multiaccounts/keypairs/database.go [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+func (kp *KeyPairs) updateKeycardLocked(kcUID string, locked bool) (err error) {
+	update, err := kp.db.Prepare(`
+		UPDATE keypairs 
+		SET 
+			keycard_locked = ?
+		WHERE 
+			keycard_uid = ?
+	`)
+	_, err = update.Exec(locked, kcUID)
```
Comment (richard-ramos) [inline multiaccounts/keypairs/database.go:178]:
```suggestion
        if err != nil {
            return err
        }
        defer update.Close()
	_, err = update.Exec(locked, kcUID)
```

File: multiaccounts/keypairs/database.go [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+	return kp.updateKeycardLocked(kcUID, false)
+}
+
+func (kp *KeyPairs) DeleteKeyPair(kcUID string) (err error) {
+	delete, err := kp.db.Prepare(`
+		DELETE FROM keypairs 
+		WHERE 
+			keycard_uid = ?
+	`)
+	_, err = delete.Exec(kcUID)
```
Comment (richard-ramos) [inline multiaccounts/keypairs/database.go:204]:
```suggestion
        if err != nil {
            return err
        }
        defer delete.Close()
	_, err = delete.Exec(kcUID)
```

**Review by richard-ramos — APPROVED** [review]

**Review by 0x-r4bbit — APPROVED** [review]

**Comment by 0x-r4bbit** [comment]
Need to get the tests green though

**Comment by 0x-r4bbit** [comment]
Restarted the tests. They are green now.

**Review by Samyoul — APPROVED** [review]

**Pull request merged**